### PR TITLE
no more "members"!

### PR DESCRIPTION
**Pull request by ahsu1230**
- Have library header filter when you click on "+4 followers" or "+5 collaborators"
- Have invites show up now (to email or to user) in manage members - divides based on whether the invite was `read_only` vs. `read_write`
- remove ability to manage members on edit library (collab experiment only)

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/libraries/libraryHeader.js [hunk 1]
```diff
@@ -568,7 +577,9 @@ angular.module('kifi')
         scope.showFollowers = function () {
           $rootScope.$emit('trackLibraryEvent', 'click', { action: 'clickedViewFollowers' });
 
-          if (scope.library.owner.id === profileService.me.id) {
+          if (scope.onCollabExperiment) {
+            scope.openMembersModal('followers_only');
```
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/angular/src/libraries/libraryHeader.js:581]: open new modal only on `collaborative` experiment

File: kifi-backend/modules/shoebox/angular/src/libraries/libraryHeader.js [hunk 2]
```diff
@@ -620,16 +632,25 @@ angular.module('kifi')
...
         };
 
+        function onWinResize() {
+          if ($window.innerWidth <= smallWindowLimit && !smallWindow) {
+            smallWindow = true;
+          } else if ($window.innerWidth > smallWindowLimit && smallWindow) {
+            smallWindow = false;
+          }
+          scope.$apply(function() {
+            setFollowers(smallWindow);
```
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/angular/src/libraries/libraryHeader.js:642]: if window is too small, show no followers (set `numFollowersToFit=0`) and make sure the number of followers changes accordingly

File: kifi-backend/modules/shoebox/angular/src/libraries/libraryHeader.js [hunk 3]
```diff
@@ -620,16 +632,25 @@ angular.module('kifi')
           });
         };
 
+        function onWinResize() {
+          if ($window.innerWidth <= smallWindowLimit && !smallWindow) {
+            smallWindow = true;
+          } else if ($window.innerWidth > smallWindowLimit && smallWindow) {
+            smallWindow = false;
+          }
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/libraries/libraryHeader.js:640]: Boolean expressions are the bomb. Five lines become one:

``` js
smallWindow = $window.innerWidth <= smallWindowLimit;
```

**Comment by 2is10** [comment]
:+1:

**Pull request merged**